scheduler/cmd/scheduler/main_test.go

```diff
@@ -1,19 +1,18 @@
-///*
-//Copyright 2020 The Kubernetes Authors.
+// /*
+// Copyright 2020 The Kubernetes Authors.
```

Comment:
should we change the header?

Reply:
I left this file there as a reference on how to test the scheduler, everything is commented out and nothing should stay eventually, but I am not sure this is the right pr to correct it

Reply:
I agree, this should be a follow up PR :)